torchaudio/__init__.py

```diff
@@ -18,29 +14,48 @@ def check_input(src):
         raise TypeError('Expected a CPU based tensor, got %s' % type(src))
 
 
-def load(filepath, out=None, normalization=None, num_frames=-1, offset=0):
+def load(filepath,
+         out=None,
+         normalization=True,
+         channels_first=True,
```

Comment:
This is not documented below. 

Also having `True` as default parameter breaks some transforms:

- [Example in MEL2()](https://github.com/dhpollack/audio/blob/sox_effects_test/torchaudio/transforms.py#L323): LC2CL() could be removed
- [DownmixMono()](https://github.com/dhpollack/audio/blob/sox_effects_test/torchaudio/transforms.py#L124) should probably also get a channels_first parameter that defaults to the same value

Reply:
This sent me down a bit of a rabbit hole.  I changed the MEL2 example and added `channels_first` to both `DownmixMono` and `PadTrim`.  The rabbit hole is that I changed the documentation to the new pytorch documentation style.  Once I get everything in order I'll post the commit here with the changes